internal/tiltfile/tiltfile.go

```diff
@@ -222,7 +222,9 @@ func (tfl tiltfileLoader) Load(ctx context.Context, filename string, userConfigS
 	tlr.UpdateSettings = us
 
 	duration := time.Since(start)
-	s.logger.Infof("Successfully loaded Tiltfile (%s)", duration)
+	if tlr.Error == nil {
```

Comment:
Haha, this has annoyed me for so long. It never occurred to me that it was actually as simple as this.